lib/ftu/filters.dart

```diff
@@ -1,25 +1,145 @@
+
+import'../ftu/parsing.dart';
+import 'dart:async';
+import 'dart:convert';
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class FiltersPage extends StatefulWidget {
+  FiltersPage({Key key}) : super(key: key);
+
+  @override
+  FiltersState createState() => FiltersState();
+}
+
+class FiltersState extends State<FiltersPage> {
+  List industryList = List();
+  List specialtyList = List();
+  List tempList = List();
+  String _industry;
+  String _specialty;
+
+  void setIndustry(newVal) {
+    setState(() {
+      _specialty = null;
+      _industry = newVal;
+      tempList = specialtyList
+          .where((x) => x.industryId.toString() == (_industry.toString()))
+          .toList();
+    });
+  }
+
+  Future<String> loadStatesProvincesFromFile() async {
+    return await rootBundle.loadString('json/filter.json');
+  }
+
+  Future<String> _populateDropdown() async {
+    String getPlaces = await loadStatesProvincesFromFile();
+    final jsonResponse = json.decode(getPlaces);
 
-// TODO: Update this placeholder widget
-class Filters extends StatelessWidget {
-  const Filters({Key key}) : super(key: key);
+    Localization places = new Localization.fromJson(jsonResponse);
+
+    setState(() {
+      industryList = places.industry;
+      specialtyList = places.specialty;
+    });
+  }
 
   @override
+  void initState() {
+    super.initState();
+    this._populateDropdown();
+  }
+
   Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: Colors.black,
-      appBar: new AppBar(
-        title: new Text("Select up to 3 options", style: TextStyle(color: Colors.white, fontSize: 20.0))
+    return Stack(children: <Widget>[
+      Image.asset(
+        "images/DefaultBg.png",
+        height: MediaQuery.of(context).size.height,
+        width: MediaQuery.of(context).size.width,
+        fit: BoxFit.fill,
       ),
-      body: InkWell(
-        onTap: (){
-          Scaffold.of(context);
+      Scaffold(
+          backgroundColor: Colors.transparent,
+          appBar: AppBar(
+            backgroundColor: Colors.transparent,
+            elevation: 0.0,
+          ),
+          body: Center(
+              child: Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: <Widget>[
+              buildTitle(),
+              buildIndustry(),
+              buildSpecialty()
+            ],
+          )),
+          floatingActionButton: FloatingActionButton(
+            backgroundColor: Colors.teal,
+            child: Text("Next"),
```

Comment:
Move all strings to a constants file. Refer to the other files in the `constants` folder for reference.

Reply:
Aysha, if you need help with constants, I can help.